src/cljs/rems/administration/organization.cljs

```diff
@@ -61,35 +61,39 @@
-     [:div.col.commands
-      [administration/back-button "/administration/organizations"]
-      [roles/show-when roles/+admin-write-roles+ ;; TODO doesn't match the API roles exactly
-       [edit-button id]
-       [status-flags/enabled-toggle organization #(rf/dispatch [:rems.administration.organizations/set-organization-enabled %1 %2 [::enter-page id]])]
-       [status-flags/archived-toggle organization #(rf/dispatch [:rems.administration.organizations/set-organization-archived %1 %2 [::enter-page id]])]]])])
+  (let [organization-id (:organization/id organization)
+        org-owner? (->> @(rf/subscribe [:owned-organizations])
+                        (some (comp #{organization-id} :organization/id)))]
+    [:div.spaced-vertically-3
+     [collapsible/component
+      {:id "organization"
+       :title (get-in organization [:organization/name language])
+       :always [:div
+                [inline-info-field (text :t.administration/id) (:organization/id organization)]
+                (doall (for [[langcode localization] (:organization/short-name organization)]
+                         ^{:key (str "short-name-" (name langcode))}
+                         [inline-info-field (str (text :t.administration/short-name)
+                                                 " (" (str/upper-case (name langcode)) ")")
+                          localization]))
+                (doall (for [[langcode localization] (:organization/name organization)]
+                         ^{:key (str "name-" (name langcode))}
+                         [inline-info-field (str (text :t.administration/title)
+                                                 " (" (str/upper-case (name langcode)) ")")
+                          localization]))
+                [inline-info-field (text :t.administration/owners) (->> (:organization/owners organization)
+                                                                        (map enrich-user)
+                                                                        (map :display)
+                                                                        (interpose [:br]))]
+                [review-emails-field (:organization/review-emails organization)]
+                [inline-info-field (text :t.administration/active) [readonly-checkbox {:value (status-flags/active? organization)}]]]}]
+     (let [id (:organization/id organization)]
+       [:div.col.commands
+        [administration/back-button "/administration/organizations"]
+        (when org-owner?
+          [edit-button id])
+        [roles/show-when roles/+admin-write-roles+ ;; TODO doesn't match the API roles exactly
+         [status-flags/enabled-toggle organization #(rf/dispatch [:rems.administration.organizations/set-organization-enabled %1 %2 [::enter-page id]])]
+         [status-flags/archived-toggle organization #(rf/dispatch [:rems.administration.organizations/set-organization-archived %1 %2 [::enter-page id]])]]])]))
```

Comment:
Should we replace this with the check for the `org-owner?` for the other buttons as well? Since it is the same condition, isn't it?

Reply:
:+1: it's almost the same condition: `(def roles/+admin-write-roles+ #{:organization-owner :owner})` is not as strict about what organization owner means since user can be organization owner in another organization. the meaning is preserved as i think it was unintended previously that _any_ org owner could edit any organization.

Reply:
modified this to check that user must strictly be `#{:owner}` as organization owners currently cannot use enable/archive actions, as discussed